--- /etc/nginx/nginx.conf.original	2019-01-21 19:39:10.556032081 +0100
+++ /etc/nginx/nginx.conf	2019-01-21 19:42:09.225622416 +0100
@@ -45,6 +45,12 @@
         include /etc/nginx/default.d/*.conf;
 
         location / {
+          proxy_pass http://localhost:8000;
+          proxy_http_version 1.1;
+          proxy_set_header Upgrade $http_upgrade;
+          proxy_set_header Connection 'upgrade';
+          proxy_set_header Host $host;
+          proxy_cache_bypass $http_upgrade;
         }
 
         error_page 404 /404.html;
